news/2020-11-03-new-featured-artist-se-u-ra.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Featured Artist: Se-U-Ra"
+date: 2020-11-03 09:00:00 +0000
+---
+
+We're excited to welcome **Se-U-Ra** aboard as our latest addition to the Featured Artist line-up!
+
+![](https://assets.ppy.sh/artists/112/header.jpg)
+
+No stranger to the rhythm gaming scene, **Se-U-Ra** is a Japanese artist and producer who has appeared in games like *Dynamix* and *K-Shoot Mania* and is a regular participant in the *BMS* community.
+
+With a whole new spin on the sound of a scene we all know and love, we're sure you'll find something to pique your interest in this latest selection of **38** new tracks, plus a showcase map from the masterminds of the Mappers' Guild.
+
+Check it all out:
+
+### Qlifact -another-
+
+Check out [this showcase set](https://osu.ppy.sh/beatmapsets/1224699) hosted by [Nao Tomori](https://osu.ppy.sh/users/5364763) of the Mappers' Guild, including [Icekalt](https://osu.ppy.sh/users/5410645)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/112/previews/2940.mp3" type="audio/mpeg">
+</audio>
+
+### LOA2
+
+Check out [this collaborative set](https://osu.ppy.sh/beatmapsets/929622) hosted by [Azzedd](https://osu.ppy.sh/users/8889054)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/112/previews/2933.mp3" type="audio/mpeg">
+</audio>
+
+### QSHELL -Kyoshoku no Shell-
+
+Check out [this collaborative set](https://osu.ppy.sh/beatmapsets/928283) hosted by [Azzedd](https://osu.ppy.sh/users/8889054)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/112/previews/2939.mp3" type="audio/mpeg">
+</audio>
+
+### LOSHAXI -HACK-
+
+Check out [this osu!taiko set](https://osu.ppy.sh/beatmapsets/1094418) by [Genjuro](https://osu.ppy.sh/users/3196091), which was featured in the osu!taiko World Cup 2020 Semifinals!
```

Comment:
```suggestion
Check out [this osu!taiko set](https://osu.ppy.sh/beatmapsets/1094418) by [Genjuro](https://osu.ppy.sh/users/3196091), which was featured in the [osu!taiko World Cup 2020 Semifinals](/wiki/Tournaments/TWC/2020#semifinals)!
```
thought it would be nice to leave a link to the relevant wiki article here